### ISSUE TITLE
racer install failed based on $ rustc --version rustc 1.38.0-nightly (00ee1b47f 2019-08-11)

### ISSUE THREAD

**Issue by xiaods** [racer-rust/racer#1066]
it can't install . 
```
 Compiling rustc-ap-rustc_data_structures v542.0.0                  
error[E0658]: use of unstable library feature 'test': `bench` is a part of custom test frameworks which are unstable
    --> /Users/xiaods/.cargo/registry/src/mirrors.ustc.edu.cn-61ef6e0cd06fb9b8/rustc-ap-rustc_data_structures-542.0.0/bit_set.rs:1269:3                                         
     |
1269 | #[bench]
     |   ^^^^^
     |
     = note: for more information, see https://github.com/rust-lang/rust/issues/50297
     = help: add `#![feature(test)]` to the crate attributes to enable

error[E0658]: use of unstable library feature 'test': `bench` is a part of custom test frameworks which are unstable
    --> /Users/xiaods/.cargo/registry/src/mirrors.ustc.edu.cn-61ef6e0cd06fb9b8/rustc-ap-rustc_data_structures-542.0.0/bit_set.rs:1284:3
     |
1284 | #[bench]
     |   ^^^^^
     |
     = note: for more information, see https://github.com/rust-lang/rust/issues/50297
     = help: add `#![feature(test)]` to the crate attributes to enable

error[E0658]: use of unstable library feature 'test': `bench` is a part of custom test frameworks which are unstable
    --> /Users/xiaods/.cargo/registry/src/mirrors.ustc.edu.cn-61ef6e0cd06fb9b8/rustc-ap-rustc_data_structures-542.0.0/bit_set.rs:1302:3
     |
1302 | #[bench]
     |   ^^^^^
     |
     = note: for more information, see https://github.com/rust-lang/rust/issues/50297
     = help: add `#![feature(test)]` to the crate attributes to enable

error[E0658]: use of unstable library feature 'test': `bench` is a part of custom test frameworks which are unstable
    --> /Users/xiaods/.cargo/registry/src/mirrors.ustc.edu.cn-61ef6e0cd06fb9b8/rustc-ap-rustc_data_structures-542.0.0/bit_set.rs:1320:3
     |
1320 | #[bench]
     |   ^^^^^
     |
     = note: for more information, see https://github.com/rust-lang/rust/issues/50297
     = help: add `#![feature(test)]` to the crate attributes to enable

error[E0658]: use of unstable library feature 'test': `bench` is a part of custom test frameworks which are unstable
    --> /Users/xiaods/.cargo/registry/src/mirrors.ustc.edu.cn-61ef6e0cd06fb9b8/rustc-ap-rustc_data_structures-542.0.0/bit_set.rs:1335:3
     |
1335 | #[bench]
     |   ^^^^^
     |
     = note: for more information, see https://github.com/rust-lang/rust/issues/50297
     = help: add `#![feature(test)]` to the crate attributes to enable
...sliped....


```

**Comment by kngwyu** [comment]
Sorry, I'll release the new version soon.

**Comment by kngwyu** [comment]
Released 2.1.28 now.
